src/Symfony/Component/Workflow/Tests/Dumper/PlantUmlDumperTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Tests\Dumper;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Workflow\Dumper\DumperInterface;
+use Symfony\Component\Workflow\Dumper\PlantUmlDumper;
+use Symfony\Component\Workflow\Marking;
+use Symfony\Component\Workflow\Tests\WorkflowBuilderTrait;
+
+/**
+ * Class PlantUmlDumperTest.
```

Comment:
To be removed.
And IIRC the rule is to not add `@author` on Test class

Reply:
ok done.